### PR TITLE
Removed the TABLEPASS flag.

### PR DESCRIPTION
**Pull request by ComicIronic**
No longer in use in any of the code.
Also changed cases of flags being set to null instead of 0.

### CODE REVIEW

**Comment by d3athrow** [comment]
tablepass isn't used anymore?

**Comment by ComicIronic** [comment]
All the pass flags are now in the pass_flags var, which includes PASSTABLE.
Considering all I did was remove TABLEPASS from var lists, and I didn't touch any code except for a line which added it to a flags list, it seems to be totally unused.

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -305,7 +305,6 @@ var/list/DummyCache = list()
 	D.flags=initial(D.flags)
 	D.pass_flags=initial(D.pass_flags)
 	if(pass_flags&PASSTABLE)
-		D.flags      |= TABLEPASS
```
Comment (ComicIronic) [inline code/__HELPERS/game.dm]: This line is actually pretty pointless, since TABLEPASS is never checked anywhere.

**Comment by d3athrow** [comment]
ok

**Pull request merged**